src/Microsoft.IdentityModel.Tokens/TokenValidationParameters.cs

```diff
@@ -462,6 +462,10 @@ public string NameClaimType
         /// Gets or sets a value indicating whether SAML tokens must have at least one AudienceRestriction.
         /// The default is <c>true</c>.
         /// </summary>
+        /// <remarks>
+        /// If set to false and the Audience is null, Audience validation will be skipped.
+        /// If set to false and the Audience is not null, the Audience will still be validated.
```

Comment:
So it's really a demand audiences?

Reply:
🤔 I am following Brent's comment in the issue [here](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/1547#issuecomment-961428386). Please let me know if I can make it more clear!